validation/src/distributions/binom.r

```diff
@@ -0,0 +1,102 @@
+
+// Copyright (C) 2012 - Prateek Papriwal
+//
+// This file must be used under the terms of the CeCILL.
+// This source file is licensed as described in the file COPYING, which
+// you should have received as part of this distribution.  The terms
+// are also available at
+// http://www.cecill.info/licences/Licence_CeCILL_V2-en.txt

```

Comment:
I did not see that before, but the license must be added (eg COPYING.CeCILL_V2) and the files using this specific license must be mentioned in the LICENSE file

Reply:
I have added the license file to the folder containing the script. I don't think we need to put it at the forefront of the repo since, as you said, it applies to a stand-alone file unrelated to OpenTURNS code.

Reply:
but its also used in python/test/t_binomial_dataset.csv

Reply:
Not that it matters (as you said, this file is independent from OT), but CeCILL is actually compatible with the LGPL (and even the GPL for that matter): http://www.cecill.info/faq.fr.html#compatible

Reply:
you cannot include cecill code into an lgpl project, but the other way around is ok

Reply:
Ah yes indeed, you are right. I think I will simply duplicate the COPYING.CECILL_V2 file in the test directories because putting it in the main folder might give people the impression parts of OpenTURNS are licensed under CECILL.

Reply:
Yes, but all of them refer to parts of the library. The file is not very large anyway.